docs/zh/latest/plugins/response-rewrite.md

```diff
@@ -23,19 +23,21 @@ title: response-rewrite
 
 ## 目录
 
-- [**response-rewrite**](#response-rewrite)
+- [**简介**](#简介)
```

Comment:
Maybe we can remove them as we already generate the table of content on the doc site?
https://apisix.apache.org/docs/apisix/plugins/redirect

@juzhiyuan 
What about your opinion?

Reply:
Agree! We have more people using website 😉

Reply:
> Agree! We have more people using website 😉

so? remove them?and also other docs?

Reply:
Hi @liangliang4ward, I would prefer to keep this PR as current status and submit a new PR to remove all Outlines from markdown files, @liangliang4ward could you help to do that?

Reply:
i'd like to